include/inexor/vulkan-renderer/exceptions/exception.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <stdexcept>
+#include <string>
+
+namespace inexor::vulkan_renderer::exceptions {
+
+/// @brief A custom base class for exceptions
+class Exception : public std::runtime_error {
+public:
+    /// @brief
+    /// @param message
+    Exception(std::string message);
+    Exception() = delete;
+    Exception(const Exception &) = delete;
+    Exception(Exception &&) = delete;
+    ~Exception() = default;
+
+    Exception &operator=(const Exception &) = delete;
+    Exception &operator=(Exception &&) = delete;
```

Comment:
```suggestion
    using std::runtime_error::runtime_error;
```

No need to define own constructors etc. just use it from the base class.

Reply:
Done.